core/controllers/suggestion.py

```diff
@@ -119,7 +119,16 @@ def put(self, target_id, suggestion_id):
 
         action = self.payload.get('action')
         suggestion = suggestion_services.get_suggestion_by_id(suggestion_id)
+
         if action == suggestion_models.ACTION_TYPE_ACCEPT:
+            if (
```

Comment:
This is icky. Why don't you do it in the frontend before the suggestion comes in?

Reply:
So we only pass the suggestion_id to the backend, and we retrieve the suggestion object and then accept. So we cannot populate the field in the frontend. I actually removed the function I had written, and updated suggestion.change.skill_id. This will save us one write operation to the DB. I think this is better.

Reply:
OK, thanks for the explanation. I think this makes sense.